config/settings.yml

```diff
@@ -1501,7 +1501,6 @@ vanotify:
         contact_info_change: <%= ENV['vanotify__services__va_gov__template_id__contact_info_change'] %>
         covid_vaccine_registration: <%= ENV['vanotify__services__va_gov__template_id__covid_vaccine_registration'] %>
         direct_deposit: <%= ENV['vanotify__services__va_gov__template_id__direct_deposit'] %>
-        direct_deposit_comp_pen: <%= ENV['vanotify__services__va_gov__template_id__direct_deposit_comp_pen'] %>
```

Comment:
Can you make sure to delete this from AWS Parameter store (for all envs) after this PR is _deployed_? Let me know if you need me to do it for you.